adminSiteClient/ChartEditor.ts

```diff
@@ -201,6 +205,44 @@ export class ChartEditor extends AbstractChartEditor<ChartEditorManager> {
             )
     }
 
+    async saveAsNarrativeView(): Promise<void> {
+        const { patchConfig, grapher } = this
+
+        const chartJson = { ...patchConfig }
+        delete chartJson.id
```

Comment:
This "getting rid of unwanted properties" for narrative views happens in at least one other place. I think it's worth pulling that out into a helper function (also for a nicer diff, see my other comment at the diff editor setup)